xc/xc7/tests/install_test/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+if(${USE_CONDA})
```

Comment:
This needs to be fixed, rather than disabled.  I expect that the use of CONDA_DIR should be removed, and `conda activate` invoked instead.

Reply:
Fixed.